packages/python/plotly/_plotly_utils/colors/_swatches.py

```diff
@@ -25,13 +38,13 @@ def _swatches(module_names, module_contents):
             for name, colors in reversed(sequences)
         ],
         layout=dict(
-            title=module_names,
+            title="plotly.colors." + module_names.split(".")[-1],
             barmode="stack",
             barnorm="fraction",
-            template="plotly",
             bargap=0.5,
             showlegend=False,
             xaxis=dict(range=[-0.02, 1.02], showticklabels=False, showgrid=False),
             height=max(600, 40 * len(sequences)),
+            template=args["template"],
```

Comment:
Could we somehow add control of the width and maybe also height parameter? I had this in the previous PR:

```
height=max(px.defaults.height, 40 * len(sequences)),
width=px.defaults.width,
```

Reply:
We could, but I'm not sure there's such a strong use-case for this... The template thing was quite important to be able to see things against the proper background etc :)

Reply:
Let's do it in another PR?